client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-overview-panel-mobile/incident-overview-panel-mobile.component.html

```diff
@@ -1,11 +1,62 @@
-
 <div class="panel">
-  <div class="content-panel">
-        <div class="content-panel-stack">
-            <h1>Wildfire Details</h1>
-            <div class="contents">
-                <ckeditor id="ck-doc" [editor]="Editor" [data]="incident.incidentOverview || '<p>For more information on this incident, please contact the information officers assigned.</p>'" (ready)="onReady($event)"></ckeditor>
+    <div class="content-panel">
+        <div class="resource-detail" *ngIf="incident.resourceDetail">
+            {{incident.resourceDetail}}
+        </div>
+        <div *ngIf="incident.responseTypeCode" class="content-panel-stack">
+            <h1>Response Type</h1>
+            <h4>{{getResponseTypeTitle(incident.responseTypeCode)}}</h4>
+            <div class="description">{{getResponseTypeDescription(incident.responseTypeCode)}}</div>
+            <hr>
+        </div>
+        <div *ngIf="incident.wildfireCrewResourcesInd || incident.wildfireAviationResourceInd || incident.incidentMgmtCrewRsrcInd || incident.heavyEquipmentResourcesInd || incident.structureProtectionRsrcInd"
+            class="content-panel-stack">
+            <h1>Resources Assigned</h1>
+            <div class="resource-panel" *ngIf="incident.incidentMgmtCrewRsrcInd">
+                <div class="subtitle"><img class="image-size"
+                        src="../../../assets/images/svg-icons/incident_management_team.svg"
+                        alt="Incident Management Team" />Incident Management Team</div>
+                <div class="contents">{{incident.incidentMgmtCrewRsrcDetail}}</div>
+                <hr>
+            </div>
+            <div class="resource-panel" *ngIf="incident.wildfireCrewResourcesInd">
+                <div class="subtitle"><img class="image-size"
+                        src="../../../assets/images/svg-icons/firefighting_personnel.svg"
```

Comment:
You should be able to set the src paths to just `/assets/images/...`. Prevents possible path issues